include/glow/IR/IR.h

```diff
@@ -320,6 +320,12 @@ class IRFunction final : public Named {
   /// \returns the variable map.
   VariableMap &getVariableMap() { return variableMap_; }
 
+  /// Returns a list of constants associated with function.
+  std::list<const Constant *> getConstants() const;
```

Comment:
Why not use `ConstList`?

Reply:
It's a list of const Constants, ConstList is a list of Constants